mathesar_ui/src/sections/table-view/actions-pane/record-operations/FilterEntries.svelte

```diff
@@ -0,0 +1,46 @@
+<script lang="ts">
+  import { createEventDispatcher } from 'svelte';
+  import { InputGroupText, Select } from '@mathesar-component-library';
+  import {
+    filterCombinations,
+    defaultFilterCombination,
+  } from '@mathesar/stores/table-data';
+  import type { FilterEntry } from '@mathesar/stores/table-data/types';
+  import type { FilterCombination } from '@mathesar/api/tables/records';
+  import FilterEntryComponent from './FilterEntry.svelte';
+  import type { ProcessedTableColumnMap } from '../../utils';
+
+  const dispatch = createEventDispatcher<{
+    remove: number;
+    update: number;
+    updateCombination: FilterCombination;
+  }>();
+
+  export let entries: FilterEntry[];
+  export let processedTableColumnsMap: ProcessedTableColumnMap;
```

Comment:
For code readability and reduced mental friction, I'd ideally like to see consistent pluralization of "columns" across `ProcessedTableColumnMap` and `processedTableColumnsMap`

Reply:
I agree. Seems like I didn't pay much attention while naming the interface.